lib/Target.js

```diff
@@ -106,11 +106,11 @@ class Target {
   }
 
   /**
-   * @return {"page"|"background_page"|"service_worker"|"shared_worker"|"other"|"browser"}
+   * @return {"page"|"background_page"|"service_worker"|"shared_worker"|"other"|"browser"|"iframe"}
```

Comment:
How bad does it look if we don't add this?
I don't think any puppeteer user should ever connect to an oopif directly. Can we just use _targetInfo.type in the test?

Reply:
Sure! Closing this in favor of https://github.com/GoogleChrome/puppeteer/pull/4516